pkg/workloads/cortex/tf_api/api.py

```diff
@@ -442,6 +442,24 @@ def get_signature(app_name, api_name):
     return jsonify(response)
 
 
+def download_dir_external(ctx, s3_path, local_path):
+    util.mkdir_p(local_path)
+    bucket_name, prefix = ctx.storage.deconstruct_s3_path(s3_path)
+    objects = ctx.storage.list_objects(s3_path)["Contents"]
+    version = prefix.split("/")[-1]
+    for obj in objects:
+        local_key = obj["Key"].lstrip(prefix[: -len(version)])
+        if not os.path.exists(os.path.dirname(local_key)):
+            util.mkdir_p(os.path.join(local_path, os.path.dirname(local_key)))
+
+        if obj["Key"][-1] == "/":
```

Comment:
Is this check necessary? I wouldn't expect directories to be returned by `list_objects()`.

Reply:
Yes, list objects would include top level dir